apps/api/src/expenses/expenses.service.ts

```diff
@@ -122,4 +130,66 @@ export class ExpensesService {
     const [result] = await this.prisma.$transaction([writeExpense, writeVault])
     return result
   }
+
+  async create_expense_file(
+    expenseId: string,
+    mimetype: string,
+    filename: string,
+    // uploadedBy: string,
+    buffer: Buffer,
+  ): Promise<string> {
+    const file: CreateExpenseFileDto = {
+      filename,
+      mimetype,
+      expenseId,
+      uploaderId: '9b26b247-c275-4320-9831-fb7a0d243758' /*TODO: get user id from context*/,
+    }
```

Comment:
this fails when testing locally - the easiest is to get it from @AuthenticatedUser() annotation - see campaign-file.controller.ts line 43

Reply:
Sorry about this. fixed it already, but it was not pushed.